ftl/ftl_node_benchmark_yaml.py

```diff
@@ -19,9 +24,28 @@
     default=5,
     help='Number of times to build the image.')
 
+parser.add_argument(
+    '--dep-test',
+    dest='dep_test',
+    action='store_true',
+    default=False,
+    help='Flag to enable to dependency test for the benchmark.')
+
+parser.add_argument(
+    '--app-size-test',
+    dest='app_size_test',
+    action='store_true',
+    default=False,
+    help='Flag to enable the app size test for the benchmark.')
+
+parser.add_argument('-b', action='store_true', default=False)
+
 
 def main():
     args = parser.parse_args()
+    if not args.dep_test and args.app_size:
```

Comment:
Parens after the "not" would make this a bit more clear.

Reply:
I guess the idea is to make it run both if you don't pass either?

Reply:
yea, that's the idea. parens added